styles/page_interfaces.less

```diff
@@ -458,3 +458,59 @@
         min-height: 10px;
     }
 }
+
+/* ------------------------------------------------------------------------ */
+/* Control Pane */
+
+.control-panel {
```

Comment:
Please rename this `control-bar` to reflect the JS naming, also the comment two lines up.